styles/index.css

```diff
@@ -0,0 +1,162 @@
+.pagecontainer {
+  display: flex;
+  width: 100%;
+  justify-content: center;
+  background-color: #CDD6DD;
+}
+
+.container {
+  display: flex;
+  flex-flow: column wrap;
+  width: 80%;
+  justify-content: center;
+  background-color: #CD8987;
+  padding-top: 2em;
+  font-family: 'Roboto', sans-serif;
+}
+
+nav ul {
+  display: grid;
+  grid-template-columns: 70% 15% 15%;
+  font-weight: bold;
+  font-family: Arial, bolder, sans-serif;
+  margin: auto;
+  font-size: 1.1em;
+}
+
+.splash {
+  display: grid;
+  grid-template-rows: 50% 20% 30%;
+  max-height: 600px;
+  justify-content: center;
+  padding-bottom: 8em;
+}
+
+h1 {
+  font-size: 5em;
+  line-height: .5em;
+  text-align: center;
+}
+
+.splash sub {
+  height: 2em;
+  font-size: 2em;
+  font-weight: bold;
+  text-transform: capitalize;
+  font-stretch: ultra-expanded;
+}
+
+.splash a[href], a:active {
+  background-color: #CDD6DD;
+  border: 2px solid black;
+  border-radius: 10px;
+  margin: 0em 4.5em 0em 4.5em;
+  line-height: 3em;
+  font-size: 1.7em;
+  vertical-align: middle;
+  text-align: center;
+  text-decoration: none;
+  color: black;
+}
+
```

Comment:
This is such a cool thing you did here and a great way to showcase different ways of approaching targeting items.